src/drive/web/modules/views/Public/index.jsx

```diff
@@ -168,20 +168,22 @@ const PublicFolderView = ({
       <Main isPublic={true}>
         <ModalStack />
         <ModalManager />
-        <PublicToolbar
-          files={files}
-          hasWriteAccess={hasWritePermissions}
-          refreshFolderContent={refreshFolderContent}
-        />
         <SharingBanner />
         <div className="u-pt-2">
           <FolderViewHeader>
             {currentFolderId && (
-              <FolderViewBreadcrumb
-                getBreadcrumbPath={geTranslatedBreadcrumbPath}
-                currentFolderId={currentFolderId}
-                navigateToFolder={navigateToFolder}
-              />
+              <>
+                <FolderViewBreadcrumb
+                  getBreadcrumbPath={geTranslatedBreadcrumbPath}
+                  currentFolderId={currentFolderId}
+                  navigateToFolder={navigateToFolder}
+                />
+                <PublicToolbar
+                  files={files}
+                  hasWriteAccess={hasWritePermissions}
+                  refreshFolderContent={refreshFolderContent}
+                />
```

Comment:
So we only display this publicToolbar if we are in a folder, not when we have a direct access to the file?

Reply:
C'est ça. Dans un accès direct au fichier il y a le viewer, qui a sa propre toolbar.